app/components/public/add-to-calender.ts

```diff
@@ -71,4 +91,12 @@ export default class AddToCalender extends Component<Args> {
     return [{ name: 'Google Calendar', url: this.googleUrl }, { name: 'iCal', url: this.iCalUrl }, { name: 'Yahoo', url: this.yahooUrl }, { name: 'Outlook', url: this.outlookUrl }];
   }
 
+  get SessioniCalUrl(): string {
```

Comment:
Why capital? Is it class name?

I thought you got the idea from my last review? Or by seeing that no other method name startss from capital letter

Reply:
oh sorry sir